private/itest.bzl

```diff
@@ -362,6 +364,21 @@ _service_test_attrs = {
     "env": attr.string_dict(
         doc = "The service manager will merge these variables into the environment when spawning the underlying binary.",
     ),
+    "allow_svcctl": attr.bool(
+        default = False,
+        doc = """If true, the test runner can access service control HTTP server at port SVCCTL_PORT.
+
+The HTTP server listens to `http://127.0.0.1:{SVCCTL_PORT}`. It is useful for tests that need to
+start / stop services in the midst of the test run. There are currently 4 API endpoint available.
+All of them are GET requests:
```

Comment:
it seems a bit odd to make them GET, especially `kill` :) perhaps they should be POST and the params go in the body? that would also avoid having the label (which includes slashes) in the path, which is probably fine but might get escaped/ugly. Was there a reason for preferring the GET?

Reply:
The only reason that this is GET is that for most languages, it is MUCH easier to send GET request than to send POST request with data in the request body. Practically, it doesn't make a difference for the service control code in the test environment, since there is no security concern.